### PR TITLE
Update OpenShift Virtualization overview dashboard

### PR DESCRIPTION
**Pull request by sradco**
- Updated the colors since the text didn't appear correctly when using light mode.
- Updated the number of VMs to show 0 if no VMs are running
- Added the `kubevirt_hco_system_health_status`, since it is important to indicate in cases where there is an issue in the conditions, but no alert.

Signed-of-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

**Comment by sradco** [comment]
/assign @philipgough 
Hi, I updated the OpenShift Virtualization dashboard, I will appreciate you review.
The fix needs to also be backported to ACM 2.11. Thank you.

**Comment by jacobbaungard** [comment]
/test test-unit

(flaky test that fails...)

**Review by jacobbaungard — APPROVED** [review]
/lgtm

**Comment by jacobbaungard** [comment]
e2e-test has unrelated problems. The tests did pass before /lgtm'ing after which it re-ran tests, merging manually.

**Comment by jacobbaungard** [comment]
Ok, cannot merge due to commits not being signed. @sradco, could you please sign-off the commit? (`git commit --amend --signoff`).

**Pull request merged**
